siad/daemon.go

```diff
@@ -24,6 +28,20 @@ func startDaemon() error {
 	// Establish multithreading.
 	runtime.GOMAXPROCS(runtime.NumCPU())
 
+	// Print a startup message.
+	fmt.Println("siad is loading")
```

Comment:
It's going to stay there until it's almost immediate. It's really painful to start siad an have no idea when it's done loading.